src/WebServer.ino

```diff
@@ -123,7 +123,7 @@ private:
     buf = "";
     if (beforeTXRam < 3000) {
       lowMemorySkip = true;
-      WebServer.send(200, "text/plain", "Low memory. Cannot display webpage :-(");
+      WebServer.send(200, F("text/plain"), "Low memory. Cannot display webpage :-(");
```

Comment:
WebServer.send(200, F("text/plain"), F("Low memory. Cannot display webpage :-("));

Reply:
You're right :)
This was just a commit to save my work before going to sleep. I've been tracking a lot of these F-macro strings